class-tgm-plugin-activation.php

```diff
@@ -448,6 +451,11 @@ public function init() {
 			if ( true === $this->has_forced_deactivation ) {
 				add_action( 'switch_theme', array( $this, 'force_deactivation' ) );
 			}
+
+			// Display forced activation notice, if present.
+			if ( current_user_can( 'manage_options' ) && is_admin() ) {
```

Comment:
Right... the property `has_forced_activation` indicates if there are plugins that need to be force activated, where the the transient and message are displaying only if they have _already been_ force activated, but we still need to notify the user.